### PR TITLE
Improve Share the Development Database

### PR DESCRIPTION
**Pull request by MarkvanMents**
Review of documentation triggered by issue #8408

### CODE REVIEW

File: content/en/docs/howto/data-models/sharing-the-development-database.md [hunk 1]
```diff
@@ -101,23 +101,28 @@ The data is now committed to the Team Server and can be used by other team membe
...
 
-1. To update the application, click **Update** on the **Changes** tab.
+1. Click **Update** on the **Changes** tab to update the application.
 
     {{< figure src="/attachments/howto/data-models/sharing-the-development-database/18580419.png" class="no-border" >}}
 
-2. To implement the data snapshot, extract the database from a ZIP file to the **deployment** directory.
+1. Click **App** > **Show App Directory in Explorer** to open the root directory of your app.
+
+    The data snapshot is in the file `datasnapshot.zip`.
```
Comment (basdebakker) [inline content/en/docs/howto/data-models/sharing-the-development-database.md]: The name is `data-snapshot.zip`.

File: content/en/docs/howto/data-models/sharing-the-development-database.md [hunk 2]
```diff
@@ -101,23 +101,28 @@ The data is now committed to the Team Server and can be used by other team membe
...
+1. Click **Update** on the **Changes** tab to update the application.
 
     {{< figure src="/attachments/howto/data-models/sharing-the-development-database/18580419.png" class="no-border" >}}
 
-2. To implement the data snapshot, extract the database from a ZIP file to the **deployment** directory.
+1. Click **App** > **Show App Directory in Explorer** to open the root directory of your app.
+
+    The data snapshot is in the file `datasnapshot.zip`. 
+
+1. Extract the database from the ZIP file to the **deployment** directory.
```
Comment (basdebakker) [inline content/en/docs/howto/data-models/sharing-the-development-database.md]: Better extract it somewhere else. If you extract to the `deployment` directory, the files end up in `deployment/data`, which the next step will instruct you to delete.

File: content/en/docs/howto/data-models/sharing-the-development-database.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/howto/data-models/sharing-the-development-database.md]: Yes, don't know why I missed the hyphen 🤦

File: content/en/docs/howto/data-models/sharing-the-development-database.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/howto/data-models/sharing-the-development-database.md]: Updated to extract it somewhere temporary to start with!

**Comment by MarkvanMents** [comment]
Approved in Slack.

**Pull request merged**